src/blocks/time.rs

```diff
@@ -107,6 +111,11 @@ pub async fn run(config: &Config, api: &CommonApi) -> Result<()> {
 
         api.set_widget(widget)?;
 
+        let phase = Utc::now().second() as u64 % config.interval.seconds();
```

Comment:
`Utc::now()` is already called a few lines above, let's reuse that value.

Reply:
Done, I've also reused the `config.interval.seconds()` value